packages/sit-onyx/src/components/OnyxTable/types.ts

```diff
@@ -9,4 +9,11 @@ export type OnyxTableProps = DensityProp & {
    * If `true`, vertical borders will be displayed in addition to the default horizontal borders.
    */
   withVerticalBorders?: boolean;
+  /**
+   * Whether the table will be displayed at it's full size and is scrollable only in the page context.
```

Comment:
```suggestion
   * Whether the table will be displayed at it's full size and is scrollable only in the page context.
```

I am not sure about the wording "display at its full size" that could be misleading because people the property only affects scrolling and not size